README.md

```diff
@@ -1,11 +1,23 @@
 oerp-pricelist
 ==============
 
-Export HTML pricelist from OERP
+Export a HTML pricelist from OERP
 
-prerequisites:
+setup
+-----
 
-```sudo pip install oerplib```
-```apt-get install python-repoze.lru python-natsort```
+```sh
+# If you prefer apt packages:
+apt-get install python-repoze.lru python-natsort
```

Comment:
It's here: https://github.com/fau-fablab/oerp-pricelist/blob/ba903a962352e3fedcbbb5716735303f3fa2ef17/requirements.txt#L1